src/zql/ivm/source/set-source.ts

```diff
@@ -120,12 +119,15 @@ export abstract class SetSource<T> implements Source<T> {
     this._materialite.addDirtySource(this.#internal);
     return this;
   }
+
+  get(key: T): T | null {
```

Comment:
Can we use `undefined` for non present keys. That would be more consistent with everything else.